src/utils/getDocumentHeight.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Get the height of the document
+ *
+ * @returns {documentHeight: number}
+ */
+export default function (doc) {
+  return Math.max(
+    doc.documentElement.offsetHeight || 0,
+    doc.height || 0,
+    doc.body.scrollHeight || 0,
+    doc.body.offsetHeight || 0
+  );
```

Comment:
`document.height` is `undefined` on Chrome so this always returned `NaN`. This is actual code in React-Bootstrap and I don't know why it isn't broken.